jupyterhub/handlers/login.py

```diff
@@ -105,6 +105,8 @@ def _render(self, login_error=None, username=None):
                     'next': self.get_argument('next', ''),
                 },
             ),
+            "authenticator_request_otp": self.settings['request_otp'],
```

Comment:
Here's a question - should the login form template get the `authenticator` itself? Seems like it might be useful for custom login forms, but is a bit of a can of worms supported-API-wise.